SQL/players2.sql

```diff
@@ -107,6 +107,7 @@ CREATE TABLE client (
     warnbans       INTEGER,
 	usewmp         INTEGER,
     usenanoui      INTEGER,
+	tooltips       INTEGER,
```

Comment:
Non-zero values should be truncated to 1, unless Lummox wrote his own ANSI SQL implementation.